grakn-graql/src/main/java/ai/grakn/graql/internal/query/analytics/AbstractCentralityQuery.java

```diff
@@ -62,6 +62,8 @@ void initSubGraph() { //TODO: REMOVE THIS METHOD
         includeAttribute = true;
     }
 
+    //TODO: @Jason. This is only ever used by a test. Is it part of the public api? Why do we need this method?
+    @SuppressWarnings("unused")
     public V of(String... ofTypeLabels) {
         return of(Arrays.stream(ofTypeLabels).map(Label::of).collect(Collectors.toSet()));
     }
```

Comment:
It's part of the public API. The idea was to have two methods for specifying types (subgraph), one takes string, one takes label. I can remove one of the two if you want.